README.md

```diff
@@ -1,66 +1,88 @@
+# SilverStripe Maintenance
 
-### The [SilverStripe Maintenance module](https://github.com/bringyourownideas/silverstripe-maintenance "Assists with the maintainence of your SilverStripe application") is reducing your maintenance related work. Currently the module provides you information about available update as well as known security issues. Further enhancements are planned.
+[![Build Status](https://api.travis-ci.org/bringyourownideas/silverstripe-maintenance.svg?branch=master)](https://travis-ci.org/bringyourownideas/silverstripe-maintenance)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/bringyourownideas/silverstripe-maintenance/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/bringyourownideas/silverstripe-maintenance/?branch=master)
+[![codecov](https://codecov.io/gh/bringyourownideas/silverstripe-maintenance/branch/master/graph/badge.svg)](https://codecov.io/gh/bringyourownideas/silverstripe-maintenance)
 
-* Provides information about
- * available updates for composer packages,
- * known security issues of all installed packages, even dependencies of dependencies and
- * complete list of installed composer packages, including global packages and dependencies.
-* All information will be saved to the database as well as displayed in a model admin.
-* Scheduling of updates of the information
-
-
-## Source of the information
+## Overview
 
-The information is based on your composer files. So you need to have them available in the environment you plan to use this module. The modules below process the content of the composer files and check in suitable sources for information regarding your set up.
-
-The main functionality comes from these modules:
-
-* [SilverStripe Composer Security Checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker "Check your SilverStripe application for security issues")
-* [SilverStripe Composer Update Checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker "Check your SilverStripe application for available updates of dependencies.")
-* [SilverStripe Composer Versions](https://github.com/bringyourownideas/silverstripe-composer-versions "Provides your installed composer versions within your SilverStripe app, for review or other use cases.")
+The [SilverStripe Maintenance module](https://github.com/bringyourownideas/silverstripe-maintenance "Assists with the 
+maintenance of your SilverStripe application") reduces your maintenance related work.
 
+The module provides information about:
+* available updates for composer packages,
+* known security issues of all installed packages, even dependencies of dependencies and
+* complete list of installed composer packages, including global packages and dependencies.
+* All information will be saved to the database as well as displayed in a model admin.
```

Comment:
It's now shown in a report. Also do you think that we should list these here, seeing as it's satellite modules that provide these features but not this module?